### DISCUSSION TITLE
Subpages for Concepts and Tips & Tricks?

### DISCUSSION THREAD

**Discussion by florian-duffe** [CommandPost/FCPCafe#91] [FCP Cafe Questions] [answered]
I want to contribute my concept (see below), but there is not yet a category for it. I also would like to have a page for Tips & Tricks.

https://pentagonal-diamond-e75.notion.site/Let-s-call-it-Syncmaster-342bfeb1ed454bfeb4990fb08c1b9d56

**Comment by latenitefilms** [comment, marked as answer]
I've added a new **Tips & Tricks** page [here](https://github.com/CommandPost/FCPCafe/edit/main/docs/tipsandtricks.md).

Thanks!
